src/parsers/manifestjson.js

```diff
@@ -284,6 +284,13 @@ export default class ManifestJSONParser extends JSONParser {
       // properties are handled properly (e.g. we should also detect when the deprecated
       // keyword is actually used to warn the developer of additional properties not
       // explicitly defined in the schemas).
+    } else if (
+      error.instancePath === '/manifest_version' &&
+      error.keyword === 'maximum'
+    ) {
+      baseObject = messages.MANIFEST_VERSION_INVALID;
+      overrides.message = baseObject.message;
```

Comment:
Here we replace overrides.message with baseObject.message to match the rest of this function.
However, the original overrides.message is `/manifest_json must be <=3`, which gives a clearer indication of what caused the error than the baseObject.message `"manifest_version" in the manifest.json is not a valid value`.
I would suggest to keep the original overrides.message instead of replacing it with baseObject.message.
Thoughts?